src/main/java/com/axlabs/neo/grantshares/GrantSharesTreasury.java

```diff
@@ -481,15 +481,14 @@ public static void updateContract(ByteString nef, String manifest, Object data)
 
     private static void abortIfCallerIsNotOwner() {
         if (Runtime.getCallingScriptHash().toByteString() != Storage.get(getReadOnlyContext(), OWNER_KEY))
-            fireErrorAndAbort("Not authorised", "abortIfCallerIsNotOwner");
+            Helper.abort("abortIfCallerIsNotOwner" + ": " + "Not authorised");
     }
 
     private static void abortIfPaused() {
-        if (isPaused()) fireErrorAndAbort("Contract is paused", "abortIfCallerIsNotOwner");
+        if (isPaused()) Helper.abort("abortIfCallerIsNotOwner" + ": " + "Contract is paused");
     }
 
-    private static void fireErrorAndAbort(String msg, String method) {
-        error.fire(msg, method);
-        Helper.abort();
+    private static void abortWithMessage(String method, String msg) {
```

Comment:
I forgot this in the review: This function looks unused like the same function in `GrantSharesGov.java`. Let's remove it.